app/src/main/java/com/nutomic/syncthingandroid/activities/FirstStartActivity.java

```diff
@@ -33,28 +34,49 @@
 import com.nutomic.syncthingandroid.R;
 import com.nutomic.syncthingandroid.SyncthingApp;
 import com.nutomic.syncthingandroid.service.Constants;
+import com.nutomic.syncthingandroid.util.ConfigXml;
 
+import java.lang.ref.WeakReference;
 import javax.inject.Inject;
 
 public class FirstStartActivity extends Activity {
 
     private static String TAG = "FirstStartActivity";
     private static final int REQUEST_COARSE_LOCATION = 141;
     private static final int REQUEST_WRITE_STORAGE = 142;
-    private static final int SLIDE_POS_LOCATION_PERMISSION = 1;
+
+    private class Slide {
+        public int layout;
+        public int dotColorActive;
+        public int dotColorInActive;
+
+        Slide (int iLayout, int iDotColorActive, int iDotColorInActive) {
```

Comment:
what's up with this iStuff? why can't you just do this.layout = layout.

Reply:
ok